services/press-fault-detection-model-service/app/models/model_loader.py

```
@@ -0,0 +1,74 @@
+from huggingface_hub import hf_hub_download
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)